project-keeper/src/main/resources/templates/.github/workflows/dependencies_update.yml

```diff
@@ -6,7 +6,7 @@ on:
     inputs:
       vulnerability_issues:
         description: "GitHub issues for vulnerable dependencies as JSONL"
-        required: false
+        required: true
```

Comment:
I have the impression, that step-by-step the implementation proceeds and pulls the rope for projects using the PK tighter.  Should we create a migration guide? E.g. 
> Projects using older versions of PK and now upgrade to version x should do the following steps: ...

This could be a page in the company wiki, as the instructions may be incomplete at any time and may change with proceeding implementation and feedback from first usage experiences.

Reply:
Created new wiki page